packages/themes/block-theme/theme.json

```diff
@@ -316,26 +362,6 @@
 					"radius": "unset"
 				}
 			},
-			"core/buttons": {
-				"spacing": {
-					"blockGap": "var(--wp--preset--spacing--50)"
-				}
-			},
-			"core/columns": {
-				"spacing": {
-					"blockGap": "var(--wp--preset--spacing--50)"
```

Comment:
This is horisontal gap between columns. Not related to the vertical block margins.